<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>CSRM Manager</string>
  </property>
  <property name="windowIcon">
   <iconset resource="MainResource.qrc">
    <normaloff>:/CS2.png</normaloff>:/CS2.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(225, 236, 255);</string>
  </property>
  <widget class="QWidget" name="MainLayout">
   <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="0,0">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="LeftWidget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <layout class="QVBoxLayout" name="LeftLayout">
       <property name="leftMargin">
        <number>8</number>
       </property>
       <property name="topMargin">
        <number>8</number>
       </property>
       <property name="rightMargin">
        <number>8</number>
       </property>
       <property name="bottomMargin">
        <number>8</number>
       </property>
       <item>
        <widget class="QWidget" name="verticalWidget" native="true">
         <property name="styleSheet">
          <string notr="true">background-color: rgba(255, 255, 255, 100);
border-radius: 20px</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="leftMargin">
           <number>6</number>
          </property>
          <property name="topMargin">
           <number>6</number>
          </property>
          <property name="rightMargin">
           <number>6</number>
          </property>
          <property name="bottomMargin">
           <number>6</number>
          </property>
          <item>
           <widget class="QPushButton" name="MenuBT1">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>40</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(196, 212, 255);
color: rgb(244, 248, 253);
border-radius: 10px</string>
            </property>
            <property name="text">
             <string>CFG 设置</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="MenuBT2">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>40</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(196, 212, 255);
color: rgb(244, 248, 253);
border-radius: 10px</string>
            </property>
            <property name="text">
             <string>CFG 安装</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="MenuBT3">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>40</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(196, 212, 255);
color: rgb(244, 248, 253);
border-radius: 10px</string>
            </property>
            <property name="text">
             <string>自定义道具添加</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="verticalWidget_2" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>1</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QVBoxLayout" name="RightLayout"/>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="MainResource.qrc"/>
 </resources>
 <connections/>
</ui>
